lucene/core/src/java/org/apache/lucene/search/similarities/AfterEffectL.java

```diff
@@ -34,11 +34,14 @@ public final double score(BasicStats stats, double tfn) {
   }
   
   @Override
+  // TODO: add explanation for tfn
+  // Currently not possible, as CheckHits.verifyExplanation fails because
+  // in case of a single sub-expl the test expects
+  // the sub-expl's score to be equal to the parent expl's score
```

Comment:
this should be possible by rebasing or merging master back, I modified CheckHits yesterday so that it allows the score to be different from the parent explanation if the explanation matches `.*, computed as .* from:`